event_routing_backends/helpers.py

```diff
@@ -0,0 +1,140 @@
+"""
+Helper utilities for event transformers.
+"""
+import logging
+from datetime import timedelta
+from urllib.parse import parse_qs, urlparse
+
+from dateutil.parser import parse
+from django.contrib.auth import get_user_model
+from isodate import duration_isoformat
+
+# Imported from edx-platform
+# pylint: disable=import-error
+from lms.djangoapps.courseware.courses import get_course_by_id
+from opaque_keys.edx.keys import CourseKey
+from student.models import anonymous_id_for_user
```

Comment:
This should be pulled from `student.api` instead, and be added to that apps `api.py` instead of pulling it directly from the model to reduce coupling.

Reply:
as `anonymous_id_for_user` belongs to the edx-platform repo, are we going to perform changes there alongside this PR?

Reply:
Yes, we should update edx-platform first.  If you open up a PR over there I can review and help land it.